packages/desktop-client/src/hooks/useTransactions.ts

```diff
@@ -109,10 +195,270 @@ export function useTransactions({
 
   return {
     transactions,
+    runningBalances,
     isLoading,
     ...(error && { error }),
     reload,
     loadMore,
     isLoadingMore,
   };
 }
+
+type UsePreviewTransactionsResult = {
+  data: ReadonlyArray<TransactionEntity>;
+  isLoading: boolean;
+  error?: Error;
+};
+
+export function usePreviewTransactions(): UsePreviewTransactionsResult {
```

Comment:
Haven't we already got this in packages/desktop-client/src/hooks/usePreviewTransactions.ts?